nuxeo-drive-client/nxdrive/data/ui5/templates/settings-about.html

```diff
@@ -5,6 +5,10 @@
 	    <img src="imgs/github.png" />
 	    <a href="" ng-click="open_local('https://github.com/nuxeo/nuxeo-drive')">https://github.com/nuxeo/nuxeo-drive</a>
 	  </p>
+	  <p>
+	    <img src="imgs/update.png" />
+	    <a href="" ng-click="open_local('http://community.nuxeo.com/static/drive/latest')">http://community.nuxeo.com/static/drive/latest</a>
```

Comment:
You should use `Manager.get_version_finder(self, refresh_engines=True)` to be more generic and use the update site (beta or prod) configured in the platform.